downstream/modules/platform/proc-gw-add-team-organization.adoc

```diff
@@ -1,11 +1,11 @@
 :_mod-docs-content-type: PROCEDURE
 
-[id="proc-gw-add-team-organization_{context}"]
+[id="proc-gw-add-team-organization"]
 
 = Adding a team to an organization
 
 You can provide team access to an organization by adding roles to the team. To add roles to a team, the team must already exist in the organization. For more information, see xref:proc-controller-creating-a-team[Creating a team].
-To add roles for a team, the role must already exist. See xref:proc-gw-roles_auto-exec[Automation execution roles] and xref:proc-gw-roles_auto-dec[Automation decision roles] for more information about creating new roles.
+To add roles for a team, the role must already exist. See xref:proc-gw-create-roles[Creating a roles] for more information.
```

Comment:
Assuming the xref should be titled Creating roles rather than Creating a roles, right?

Reply:
Good catch!! I updated to Creating a role which is what the procedure title is.